src/library/scala/collection/mutable/ArrayBuilder.scala

```diff
@@ -520,3 +531,6 @@ object ArrayBuilder {
     override def toString = "ArrayBuilder.ofUnit"
   }
 }
+
+class ExceededMaximumArrayBuilderCapacity extends Exception(s"ArrayBuilder cannot store more than ${PStatics.VM_MaxArraySize} elements.")
```

Comment:
If we're going to have a maximum capacity exception, maybe it should be a single class that we throw for any array capacity problem?  For instance ArrayBuffer has the same issue (it throws a plain Exception).

Reply:
I'll change it to plain exception, following `ArrayBuffer`'s lead.